firebase.json

```diff
@@ -20,11 +20,11 @@
     ],
     "headers": [
       {
-        "source": "**/*.@(css|html|js|json|txt|xml|markdown|pdf)",
+        "source": "**/*.@(css|js|json|txt|xml|markdown|pdf)",
         "headers": [
           {
             "key": "Cache-Control",
-            "value": "max-age=315360000"
+            "value": "max-age=0, no-cache"
```

Comment:
@ozasadnyy @sophieH29 You can find a full `firebase.json` file config here https://firebase.google.com/docs/hosting/full-config
It depends on how long you want those files to be cached. For Hoverboard since it's controlling the cached files via Service Workers I don't think we need this advanced config.